helix-term/src/commands.rs

```diff
@@ -1753,13 +1784,56 @@ mod cmd {
         }
     }
 
+    fn paste_primary_clipboard_after(
+        cx: &mut compositor::Context,
+        _args: &[&str],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        paste_clipboard_impl(&mut cx.editor, Paste::After, ClipboardType::Selection)
+    }
+
+    fn paste_primary_clipboard_before(
+        cx: &mut compositor::Context,
+        _args: &[&str],
+        _event: PromptEvent,
+    ) -> anyhow::Result<()> {
+        paste_clipboard_impl(&mut cx.editor, Paste::After, ClipboardType::Selection)
+    }
+
+    fn replace_selections_with_primary_clipboard(
```

Comment:
You can refactor this out in combination with `replace_selections_with_clipboard` similar to the others

Reply:
Thank you for finding this!